rpc/v1/openapi.yaml

```diff
@@ -53,6 +53,7 @@ paths:
               schema:
```

Comment:
This file is going to be very user facing so wanted to make a request that you double check the swagger file looks good